android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -1835,6 +1849,11 @@ protected void onScrollChanged(int x, int y, int oldX, int oldY) {
     }
 
     protected void dispatchEvent(WebView webView, Event event) {
+      if (event.getViewTag() == RNCWebView.INVALID_VIEW_ID) {
+        FLog.w(TAG, "Unable to dispatch event: ", event.getEventName() + "due to InternalWebView not being attached.");
```

Comment:
I think a warning is enough. This will likely happen for any callbacks such as `onLoadingStarted` etc when the webview is unmounted. I am assuming it's likely we'll have the same issue on iOS. It might be nicer in the future to dispatch events for all of these callbacks so they can be listened to without a react tag.